octobot_trading/exchanges/connectors/ccxt/ccxt_connector.py

```diff
@@ -497,13 +497,13 @@ def get_fees(self, symbol):
             return {
                 enums.ExchangeConstantsMarketPropertyColumns.TAKER.value:
                     market_status.get(enums.ExchangeConstantsMarketPropertyColumns.TAKER.value,
-                                      constants.CONFIG_DEFAULT_FEES),
+                                      ) or constants.CONFIG_DEFAULT_FEES,
```

Comment:
Why ?

Reply:
When market_status[ExchangeConstantsMarketPropertyColumns.TAKER.value], market_status[ExchangeConstantsMarketPropertyColumns.MAKER.value] or market_status[ExchangeConstantsMarketPropertyColumns.FEE.value] is None, it will now properly use the fallback fees. Before my changes it would set them to None